### PR TITLE
form control style edits

### PR DESCRIPTION
**Pull request by awfrancisco**

### CODE REVIEW

File: _elements/form-controls.md [hunk 1]
```diff
@@ -52,18 +52,18 @@ lead: Form controls allow users to enter information into a page.
...
     <ul class="usa-content-list">
       <li>Avoid placeholder text for accessibility reasons. Most browsers’ default rendering of placeholder text does not provide a high enough contrast ratio.</li>
-      <li>Avoid breaking numbers with distinct sections (such as phone numbers, social security numbers, or credit card numbers) into separate input fields. For example, use one input for phone number, not three (one for area code, one for local code and one for number). Each field needs to be labeled for a screen reader and the labels for fields broken into segments are not often not meaningful.</li>
+      <li>Avoid breaking numbers with distinct sections (such as phone numbers, social security numbers, or credit card numbers) into separate input fields. For example, use one input for phone number, not three (one for area code, one for local code, and one for number). Each field needs to be labeled for a screen reader and the labels for fields broken into segments are often not meaningful.</li>
```
Comment (carodew) [inline _elements/form-controls.md]: @awfrancisco I would not have caught this earlier, but am now remembering from a previous conversation with the Writing Lab that we decided "Social Security Number" should be capitalized.

Comment (awfrancisco) [inline _elements/form-controls.md]: Yes, that sounds right. I'll edit.

**Comment by carodew** [comment]
This looks good to me. Are you still editing on this PR, or shall I merge?

**Comment by awfrancisco** [comment]
Good to merge. Moving to the next section.

**Pull request merged**